### PR TITLE
First artist services

### PR DESCRIPTION
**Pull request by OleksandrNechai**
I have tried to add 2 artist services  with tests.

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = function (karma) {
         /**
          * Filled by the task `gulp karma-conf`
          */
-        files: [          'bower_components/angular/angular.js',          'bower_components/angular-ui-router/release/angular-ui-router.js',          'bower_components/jquery/dist/jquery.js',          'bower_components/bootstrap/dist/js/bootstrap.js',          'bower_components/angular-mocks/angular-mocks.js',          '.tmp/src/app/findArtists/findArtists.module.js',          '.tmp/src/app/findArtists/find.artists.controller.test.js',          '.tmp/src/app/findArtists/find.artists.controller.js',          '.tmp/src/app/app.js',          '.tmp/musicalisimo-templates.js'        ],
```
Comment (ilyaivanov) [inline karma.conf.js]: no need to commit this

Comment (OleksandrNechai) [inline karma.conf.js]: Why not? If so why it is not among ignored files then?

File: src/app/index.html [hunk 2]
```diff
@@ -14,11 +14,13 @@
     <!-- inject:vendor:js -->
     <script src="/angular/angular.js"></script>
     <script src="/angular-route/angular-route.js"></script>
+    <script src="/underscore/sunderscore.js"></script>
```
Comment (nikitas) [inline src/app/index.html]: It's better to comment of file diff, code displays directly in conversation.

File: src/app/findArtists/find.artists.controller.js [hunk 3]
```diff
@@ -1,12 +1,25 @@
...
+    //significant, general, API
+    //details
+
+    function FindArtistsbController($scope, similarArtists) {
+        var vm = this;
+        vm.options = [];
+        vm.add = add;
+        vm.foundartists = similarArtists();
+
+        vm.foo = "123";
```
Comment (nikitas) [inline src/app/findArtists/find.artists.controller.js]: wtf is 123 ?

File: src/app/index.html (hunk 2 above)
Comment (OleksandrNechai) [inline src/app/index.html]: will do later

**Pull request merged**